openerp/addons/base/ir/ir_model.py

```diff
@@ -240,6 +240,17 @@ def _in_modules(self):
             field.modules = ", ".join(sorted(installed_names & module_names))
 
     @api.model
+    def get_property_action(self, field_name=False, field_model=False):
```

Comment:
What I will do I will create this method in ir.property and take the model ir.model.fields from env and search for the field and return the acton, Same way I will create a model proxy for ir.property in js also to call get_property_action method
(I will do this because ir.model.fields model have nothing to do with ir.property)